tox.ini

```diff
@@ -0,0 +1,32 @@
+
+[tox]
+envlist =
+    py27
+    flake8
+    pylint
+
+[testenv]
+passenv = TRAVIS TRAVIS_*
+deps =
+    coverage
+    mock
+    coveralls
+    pytest
+commands =
+    coverage run --source=sktm,tests -m pytest -v -s .
+    coverage report -m --omit="*/tests*"
```

Comment:
Instead of running coverage for tests and then removing it from the report, it would be easier to just not run it for tests directory. You can see how it's done in skt